Classes/ContentReleaseManager.php

```diff
@@ -144,7 +144,7 @@ private function resolveCurrentContentReleaseId(?string $currentContentReleaseId
 
     private function getAccountId(): ?string
     {
-        if ($this->securityContext->isInitialized()) {
```

Comment:
Would it be "better" to use `[...]->getAccount() instanceof Account` as condition (instead of `null` check?